src/iqm/cirq_iqm/iqm_operation_mapping.py

```diff
@@ -123,4 +123,21 @@ def map_operation(operation: Operation) -> Instruction:
             args={},
         )
 
+    if isinstance(operation, ClassicallyControlledOperation):
+        if len(operation._conditions) > 1:
+            raise OperationNotSupportedError('Classically controlled prx gates can only have one condition')
+        if isinstance(operation._sub_operation.gate, (PhasedXPowGate, XPowGate, YPowGate)):
+            return Instruction(
+                name='cc_prx',
+                qubits=tuple(qubits),
+                args={
+                    'angle_t': operation._sub_operation.gate.exponent / 2,
+                    'phase_t': operation._sub_operation.gate.phase_exponent / 2,
+                    'feedback_qubit': 'n/a',
+                    'feedback_key': 'n/a',
+                },
+            )
+
```

Comment:
Here we should raise an error that says we do not support classical control of this particular gate.